src/inputs/phoneInput/phoneInputCountryDropdownItem.jsx

```diff
@@ -70,9 +70,9 @@ class PhoneInputCountryDropdownItem extends React.Component {
         return (
             <div
                 className={containerClasses}
-                onMouseDown={this.onMouseDown}
-                onMouseEnter={this.onMouseEnter}
-                onMouseMove={this.onMouseMove}
+                onMouseDown={option.value && this.onMouseDown}
+                onMouseEnter={option.value && this.onMouseEnter}
+                onMouseMove={option.value && this.onMouseMove}
```

Comment:
Do we need to add this security here?

Reply:
so the idea here is that if the option does not have a value, then we should not treat it as a selectable option and just ignore events on it. So will be display only item.